frontend/src/metabase/visualizations/lib/settings/column.js

```diff
@@ -523,12 +524,22 @@ export const buildTableColumnSettings = ({
     widget: ChartSettingTableColumns,
     getHidden: (series, vizSettings) => vizSettings["table.pivot"],
     isValid: ([{ card, data }]) => {
+      const question = new Question(card /* metadata */);
       const columns = card.visualization_settings["table.columns"];
       const enabledColumns = columns.filter(column => column.enabled);
+
+      let query;
+
+      try {
```

Comment:
Leftover debugging code :)

Reply:
nope, I got usage of this method in Audit pages with internal query, so it would throw after our latest changes
https://github.com/metabase/metabase/commit/4acd8deb670b1102099c4340e2f135619e39bd5a